.ci/tweet-releasenotes.sh

```diff
@@ -32,17 +32,28 @@ CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
 echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
 
 cd .ci-temp/checkstyle
-LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
-                       | jq ".tag_name")
-echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
+
+curl https://api.github.com/repos/checkstyle/checkstyle/releases -o /var/tmp/cs-releases.json
+TARGET_RELEASE_INDEX=$(cat /var/tmp/cs-releases.json | \
+    jq '[.[].tag_name] | to_entries | .[] | select(.value=="checkstyle-$1") | .key')
```

Comment:
better to give better name to `$1`, use extra variable

Reply:
Changes Done